DDDEastAnglia/Views/Session/Details.cshtml

```diff
@@ -5,25 +5,18 @@
 }
 
 <h2>@Html.DisplayFor(model => model.SessionTitle)</h2>
-
-<div>
-    @if (Model.SpeakerGravatarUrl != null)
-    {
-        <img src="@Model.SpeakerGravatarUrl" alt="@Model.SpeakerName"  />
-    }
-    <a href="@Url.Action("Details", "Speaker", new { id = Model.SpeakerId })">@(string.IsNullOrWhiteSpace(Model.SpeakerName) ? Model.SpeakerUserName : Model.SpeakerName)</a>
-</div>
-
+@if (Model.ShowSpeakerDetails)
+{
+    <div>
+        @if (Model.SpeakerGravatarUrl != null)
+        {
+            <img src="@Model.SpeakerGravatarUrl" alt="@Model.SpeakerName"  />
+        }
+        <a href="@Url.Action("Details", "Speaker", new {id = Model.SpeakerId})">@(string.IsNullOrWhiteSpace(Model.SpeakerName) ? Model.SpeakerUserName : Model.SpeakerName)</a>
+    </div>
+}
 <div class="tweet-session">
-    @if (User.Identity.Name == Model.SpeakerUserName)
```

Comment:
Because if sessions are going to be anonymous then speakers don't get to tweet 'Look at my session'